### PR TITLE
Modified readme to clarify this is an unsupported beta for now

### PR DESCRIPTION
**Pull request by Kushmaro**
Some minimal changes to the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
-# Terraform Provider ElasticStack
+# Terraform Provider Elastic Stack
 
 [![Acceptance Status](https://devops-ci.elastic.co/job/elastic+terraform-provider-elasticstack+main/badge/icon?subject=acceptance)](https://devops-ci.elastic.co/job/elastic+terraform-provider-elasticstack+main/)
 
+## Use of the provider
+The Elastic Stack provider allows you to manage and configure the Elastic stack (elasticsearch, kibana, etc) as code using ```terraform```.
```
Comment (olksdr) [inline README.md]:
```suggestion
The Elastic Stack provider allows you to manage and configure the Elastic stack (Elasticsearch, Kibana, etc) as code using `terraform`.
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 
-## Using the provider
+Version ```0.1.0``` is an initial beta version of the Elastic Stack terraform provider. It is currently not yet fully suppoerted by elastic, but will be in subsequent versions.
```
Comment (olksdr) [inline README.md]:
```suggestion
Version `0.1.0` is an initial beta version of the Elastic Stack terraform provider. It is currently not yet fully supported by Elastic, but will be in the subsequent versions.
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 
-The Elasticstack provider provides the resources to interact with Elastic stack products.
+For this early release, we'd like to get feedback, bugs, and issues from early adopters. Any modifications made to resources in later versions will be communicated prior to release.
```
Comment (olksdr) [inline README.md]:
```suggestion
For this early release, we'd like to get feedback from early adopters. Any modifications made to resources in later versions will be communicated prior to release.
```

File: README.md (hunk 2 above)
Comment (Kushmaro) [reply README.md]: plural "versions" don't require "the"

**Pull request merged**